orttraining/orttraining/core/graph/gradient_builder_base.cc

```diff
@@ -293,5 +293,29 @@ std::vector<NodeDef> GradientBuilderBase::GetBiasGeluGradNodes(
   return result;
 }
 
+ArgDef GradientBuilderBase::HandleATenOpGradInput(const ArgDef& source_arg_def, const std::string& transform_func,
+                                                  std::vector<NodeDef>& output) const {
+  ArgDef target_arg_def = source_arg_def;
+  if (transform_func == "sizes()") {
```

Comment:
other than size() and size(x), do we see other patterns so far... 
I am wondering how far this solution will be able to support the various transform_func. 

Also, if we will be hitting different kinds of transform_func, I am thinking it's better to do the gradient building with python frontend... (e.g. the idea I shared with you the other day)

Reply:
Since it's model driven now and these are the only two patterns so far, so this is a temporary solution. I think we can think about the way how to do this on python frontend, then we can get rid of this solution.